test/integration/features/cmd-openshift.feature

```diff
@@ -31,6 +31,8 @@ cluster in VM provided by Minishift.
         On restart of OpenShift cluster containers are terminated, which sets FinishedAt to actual time. This value persist
         after next start of container.
      Given stdout of command "minishift ssh -- "docker inspect --format={{.State.FinishedAt}} origin"" is equal to "0001-01-01T00:00:00Z"
+#TO BE REMOVED with OpenShift v3.9. OpenShift v3.7 needs additional time after startup to get containers ready due to #2169.
```

Comment:
@agajdosi Now, 3.9.0 is default openshift version. Would you consider removing comment now?